EasyCosmos.Tests/MigrationExtensionsTests.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using EasyCosmos.Tests.TestModels;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace EasyCosmos.Tests
+{
+    public class MigrationExtensionsTests
+    {
+        [Fact]
+        public void CanFindMigrationTypes()
+        {
+            var migrations = GetType().Assembly.FindMigrations();
+            Assert.NotNull(migrations);
+            foreach (var migrationType in migrations)
+            {
+                Assert.True(migrationType.IsSubclassOf(typeof(CosmosMigration)));
+            }
+        }
+
+        [Fact]
+        public void OrdersMigrationsByNumericalOrder()
+        {
+            var migrations = new [] {
```

Comment:
Probably worth flipping these so a sorting operation needs to take place for this test to pass.

Reply:
The sorting doesn't happen when you call this method with a list of specific types or instances. This way you can create specific lists of migrations, such as a pre execution list and/or a post execution list. 
Though this does raise the problem that this test is a false positive and actually unnecessary